HopsanGUI/Widgets/HcomWidget.cpp

```diff
@@ -39,7 +39,7 @@
 #include <QGridLayout>
 #include <QTextBlock>
 #include <QScrollBar>
-
+#include <QAbstractItemView>
 
```

Comment:
Is this the correct include?

Reply:
Yes, it is needed for the QCompleter for some reason.